service/routes.go

```diff
@@ -0,0 +1,20 @@
+package service
+
+import (
+	"github.com/gorilla/mux"
+	"github.com/rancher/go-rancher/api"
+	"github.com/rancher/go-rancher/client"
+	//This should be v2, supporting schemas
+	"net/http"
+)
+
+var schemas *client.Schemas
+var router *mux.Router
+
+func NewRouter() *mux.Router {
+	schemas = &client.Schemas{}
+	router = mux.NewRouter().StrictSlash(true)
+	router.Methods("POST").Path("/v2-beta/webhook/construct").Handler(api.ApiHandler(schemas, http.HandlerFunc(ConstructPayload)))
```

Comment:
what is the difference between `/v1-webhooks/execute` and `/v1-webhooks-execute` in terms of authentication?